Errors corrections from previous merge

Error:(43, 67) error: cannot find symbol variable centerLanguage
Error:(44, 67) error: cannot find symbol variable languageLineOne
Error:(45, 69) error: cannot find symbol variable languageLineTwo

after clicking Setting icon in top right corner :

E/AndroidRuntime: FATAL EXCEPTION: main
                 Process: com.example.android.rpgquiz, PID: 19121
                 java.lang.IllegalStateException: Could not execute method for android:onClick
                     at android.support.v7.app.AppCompatViewInflater$DeclaredOnClickListener.onClick(AppCompatViewInflater.java:293)
                     at android.view.View.performClick(View.java:6256)
                     at android.view.View$PerformClick.run(View.java:24701)
                     at android.os.Handler.handleCallback(Handler.java:789)
                     at android.os.Handler.dispatchMessage(Handler.java:98)
                     at android.os.Looper.loop(Looper.java:164)
                     at android.app.ActivityThread.main(ActivityThread.java:6541)
                     at java.lang.reflect.Method.invoke(Native Method)
                     at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:240)
                     at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:767)
                  Caused by: java.lang.reflect.InvocationTargetException
                     at java.lang.reflect.Method.invoke(Native Method)
                     at android.support.v7.app.AppCompatViewInflater$DeclaredOnClickListener.onClick(AppCompatViewInflater.java:288)
                     at android.view.View.performClick(View.java:6256)
                     at android.view.View$PerformClick.run(View.java:24701)
                     at android.os.Handler.handleCallback(Handler.java:789)
                     at android.os.Handler.dispatchMessage(Handler.java:98)
                     at android.os.Looper.loop(Looper.java:164)
                     at android.app.ActivityThread.main(ActivityThread.java:6541)
                     at java.lang.reflect.Method.invoke(Native Method)
                     at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:240)
                     at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:767)
                  Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'void android.widget.Button.setVisibility(int)' on a null object reference
                     at com.example.android.rpgquiz.MainActivity.languageSelector(MainActivity.java:47)
                     at java.lang.reflect.Method.invoke(Native Method)
                     at android.support.v7.app.AppCompatViewInflater$DeclaredOnClickListener.onClick(AppCompatViewInflater.java:288)
                     at android.view.View.performClick(View.java:6256)
                     at android.view.View$PerformClick.run(View.java:24701)
                     at android.os.Handler.handleCallback(Handler.java:789)
                     at android.os.Handler.dispatchMessage(Handler.java:98)
                     at android.os.Looper.loop(Looper.java:164)
                     at android.app.ActivityThread.main(ActivityThread.java:6541)
                     at java.lang.reflect.Method.invoke(Native Method)
                     at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:240)
                     at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:767)

--- a/app/src/main/java/com/example/android/rpgquiz/MainActivity.java
+++ b/app/src/main/java/com/example/android/rpgquiz/MainActivity.java
@@ -5,7 +5,6 @@ import android.os.Bundle;
 import android.support.v7.app.AppCompatActivity;
 import android.view.View;
 import android.widget.Button;
-import android.widget.LinearLayout;
 import android.widget.TextView;
 
 public class MainActivity extends AppCompatActivity {
@@ -40,14 +39,8 @@ public class MainActivity extends AppCompatActivity {
         next.setVisibility(View.GONE);
 
         //language display
-        LinearLayout languageSelectionOne = this.findViewById(R.id.centerLanguage);
-        LinearLayout languageSelectionTwo = this.findViewById(R.id.languageLineOne);
-        LinearLayout languageSelectionThree = this.findViewById(R.id.languageLineTwo);
         Button languageReturn = this.findViewById(R.id.languageReturn);
 
-        languageSelectionOne.setVisibility(View.VISIBLE);
-        languageSelectionTwo.setVisibility(View.VISIBLE);
-        languageSelectionThree.setVisibility(View.VISIBLE);
         languageReturn.setVisibility(View.VISIBLE);
 
     }

--- a/app/src/main/res/layout/activity_main.xml
+++ b/app/src/main/res/layout/activity_main.xml
@@ -15,8 +15,7 @@
         android:layout_alignParentEnd="true"
         android:layout_alignParentRight="true"
         android:layout_margin="@dimen/marginOptions"
-        android:background="@drawable/option_"
-        android:onClick="languageSelector" />
+        android:background="@drawable/option_" />
 
     <Button
         android:id="@+id/logo"

